### PR TITLE
.cirrus.yml: do not install Packer from Homebrew

### PR DESCRIPTION
**Pull request by edigaryev**
It will be installed by the `mac-automation`.

See https://github.com/cirruslabs/internal-planning/issues/45.

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -4,8 +4,28 @@ env:
...
+
+  persistent_worker:
+    labels:
+      arch: $VM_ARCH
+    resources:
+      tart-vms: 2
+
+  matrix:
+    - env:
+        VM_ARCH: arm64
```
Comment (fkorotkov) [inline .cirrus.yml:18]: We now have `dev-mini-experimental` which might screw these things. 🤔  Not sure what should we do about it.

Comment (edigaryev) [reply .cirrus.yml:18]: What do you think about explicitly including it in the matrix?

Comment (edigaryev) [reply .cirrus.yml:19]: Not sure if we need "experimental" worker here?

I've added `name:` filter in https://github.com/cirruslabs/linux-image-templates/pull/31/commits/b5fbe5caed09e5ca8baf9dfef5fed4b96b796834 to only run on `dev-mini`.

**Review by fkorotkov — APPROVED** [review]

File: .cirrus.yml [hunk 2]
```diff
@@ -3,15 +3,41 @@ env:
...
+
+task:
+  name: "Install dependencies ($WORKER_NAME)"
+
+  persistent_worker:
+    labels:
+      name: $WORKER_NAME
+      arch: $WORKER_ARCH
+    resources:
+      tart-vms: 2
```
Comment (fkorotkov) [inline .cirrus.yml:17]: can do an execution lock probably but should be fine like that.

**Pull request merged**